test/images/agnhost/webhook/patch_test.go

```diff
@@ -81,6 +81,36 @@ func TestPatches(t *testing.T) {
 				},
 			},
 		},
+		{
+			patch: podsSidecarPatch,
+			initial: corev1.Pod{
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Image:     "image1",
+							Name:      "container1",
+							Resources: corev1.ResourceRequirements{},
+						},
+					},
+				},
+			},
+			expected: &corev1.Pod{
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Image:     "image1",
+							Name:      "container1",
+							Resources: corev1.ResourceRequirements{},
+						},
+						{
+							Image:     "k8s.gcr.io/pause:3.1",
```

Comment:
I think you want to use GetPauseImageName() instead of hardcoding, eg: https://github.com/kubernetes/kubernetes/blob/81acde7adc3a94dab76aea9e4df4f38c6013c2e1/test/utils/image/manifest.go#L278

Reply:
that won't work from inside the agnhost pod, right?